### ISSUE TITLE
Broken link for 'input purposes' on 1.3.5 Identify Input Purpose Understanding page

### ISSUE THREAD

**Issue by andrewhick** [w3c/wcag#2962]
On the [Understanding page for 1.3.5 Identify Input Purpose](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose), the link to 'Input Purposes for user interface components section' is broken and does not do anything.

The link's current destination is: https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose#input-purposes
I believe the link's destination should be: https://www.w3.org/TR/WCAG21/#input-purposes

This appears to have been broken since the look and feel of the understanding pages was updated in late 2022.

**Comment by alastc** [comment]
That's generated content so assigning to @michael-n-cooper

**Comment by ljoakley** [comment]
the proposed new link is the correct link for the purposes of this issue.

**Comment by patrickhlauke** [comment]
the actual fix will need to be done in the generated content part, so something for @iadawn

moving this from "Drafted" to "To do" as the actual fix hasn't been drafted yet

**Comment by kfranqueiro** [comment]
FWIW this is already fixed in the WCAG 2.2 docs using the new build system. Support for 2.1 in the new build system is still WIP.

**Comment by kfranqueiro** [comment]
This is resolved now that WCAG 2.1 informative docs have been republished via the new build system, made possible via https://github.com/w3c/wcag/pull/4007.
